pyproject.toml

```diff
@@ -1,10 +1,44 @@
+
 [build-system]
-requires = [
-    "setuptools >= 45",
-    "wheel",
-    "setuptools_scm[toml] >= 7.0.5"
-]
+requires = ["setuptools >= 65.4.0", "setuptools_scm[toml]>=7.0.5"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "festim"
+authors = [
+  { name="Remi Delaporte-Mathurin", email="rdelaportemathurin@gmail.com" },
+]
+license = {file = "LICENSE"}
+description = "Finite element simulations of hydrogen transport"
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+    "Natural Language :: English",
+    "Topic :: Scientific/Engineering",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
```

Comment:
Above there's `requires-python = ">=3.7"`

Reply:
ah yes sorry about this, it might have to raise the minimal required python a few versions